core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -297,12 +296,12 @@ public BooleanColumn copy() {
 
   @Override
   public void sortAscending() {
-    ByteArrays.mergeSort(data.elements());
+    data.sort(descendingByteComparator.reversed() /* or ByteComparators.NATURAL_COMPARATOR */);
```

Comment:
This is kind of a double negative. I'd prefer the `ByteComparators.NATURAL_COMPARATOR` option rather than reversing a reverse comparator

Reply:
Yeah, I'd use the natural comparators all over the place. But incrementally it made sense in a way to use whatever comparator you had there (just in case there was some specific logic -- I didn't really check them all).

I will update the PR to use those comparators.